src/ServiceBus/ServiceBus.Autorest/examples/Set-AzServiceBusNamespaceV2.md

```diff
@@ -0,0 +1,252 @@
+                                      "userAssignedIdentity": "/subscriptions/{subscriptionId}/resourceGroups/{myResourceGroup}/providers/Microsoft.ManagedIdentity/use
+                                  rAssignedIdentities/myFirstIdentity"
+                                    },
+                                    "keyName": "key4",
+                                    "keyVaultUri": "https://{KeyVaultName}.vault.azure.net",
+                                    "keyVersion": ""
+                                  }, {
+                                    "identity": {
+                                      "userAssignedIdentity": "/subscriptions/{subscriptionId}/resourceGroups/{myResourceGroup}/providers/Microsoft.ManagedIdentity/use
+                                  rAssignedIdentities/myFirstIdentity"
+                                    },
+                                    "keyName": "key5",
+                                    "keyVaultUri": "https://{KeyVaultName}.vault.azure.net",
+                                    "keyVersion": ""
+                                  }, {
+                                    "identity": {
+                                      "userAssignedIdentity": "/subscriptions/{subscriptionId}/resourceGroups/{myResourceGroup}/providers/Microsoft.ManagedIdentity/use
+                                  rAssignedIdentities/myFirstIdentity"
+                                    },
+                                    "keyName": "key6",
+                                    "keyVaultUri": "https://{KeyVaultName}.vault.azure.net",
+                                    "keyVersion": ""
+                                  }}
+Location                        : North Europe
+MetricId                        : {subscriptionId}:myNamespace
+MinimumTlsVersion               : 1.1
+Name                            : myNamespace
+PrincipalId                     :
+PrivateEndpointConnection       :
+ProvisioningState               : Succeeded
+PublicNetworkAccess             : Enabled
+RequireInfrastructureEncryption :
+ResourceGroupName               : myResourceGroup
+ServiceBusEndpoint              : https://myNamespace.servicebus.windows.net:443/
+SkuCapacity                     : 16
+SkuName                         : Premium
+SkuTier                         : Premium
+Status                          : Active
+SystemDataCreatedAt             :
+SystemDataCreatedBy             :
+SystemDataCreatedByType         :
+SystemDataLastModifiedAt        :
+SystemDataLastModifiedBy        :
+SystemDataLastModifiedByType    :
+Tag                             : {
+                                  }
+TenantId                        :
+Type                            : Microsoft.ServiceBus/Namespaces
+UpdatedAt                       : 11/21/2022 5:01:22 AM
+UserAssignedIdentity            : {
+                                  }
+ZoneRedundant                   : False
+```
+
+#Adds a new KeyVaultProperty to ServiceBus namespace `myNamespace`.
```

Comment:
Please remove "#" before the start of the sentence. Descriptions should not be bold. Please remove # from start of descriptions everywhere, only titles must have descriptions.